clickhouse_db_schema/oss_ci_benchmark_v3_materialized_views/schema.sql

```diff
@@ -82,3 +87,5 @@ SELECT
     timestamp AS timestamp
 FROM
     benchmark.oss_ci_benchmark_v3
+WHERE
+    tupleElement(benchmark, 'name') != 'sccache_stats';
```

Comment:
Note: Don't need to get `sccache_stats` into this materialized view for benchmark